.github/workflows/release.yml

```
@@ -68,11 +70,12 @@ jobs:
   build_arm_macos:
     runs-on: macos-latest
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@v6
     - run: ./res/ci/nixish_setup.sh && make bsd
-    - env:
-        MKN_GCC_PREFERRED: 1
-      run: KLOG=3 ./mkn build -dtKa "-std=c++20 -fPIC" -O 2 -gW 9 -o mkn_arm_osx
+      run: |
```

actionlint (1.7.12)

[error] key "run" is duplicated in element of "steps" section. previously defined at line:74,col:7

(syntax-check)